lib/kiwi_config.rb

```diff
@@ -124,7 +125,8 @@ def inject_extracted_files(output_location)
       FileUtils.mkdir_p(destination, mode: 01777)
       filter = "unmanaged_files_#{@name}_excludes"
 
-      @system_description.unmanaged_files.export_files_as_tarballs(destination)
+      @system_description.unmanaged_files.file_access.
+        export_files_as_tarballs(destination)
```

Comment:
Ok, so you're consolidating what was spread across the `FileScope` objects and `SystemFileUtils`, which is nice, but what's the reason of having a separate object for the file access. Couldn't that functionality be included in the `FileScope` instances directly as modules? 

That would simplify the consumer code and get rid of the `if self.is_a?(UnmanagedFilesScope) ... else ... end` conditional in `Scope#file_access`.

So 
```ruby
file_access = @system_description[scope].file_access
file_access.write_file(file, output_root_path)
# and
@system_description.unmanaged_files.file_access.export_files_as_tarballs(destination)
```

would become
```ruby
@system_description[scope].write_file(file, output_root_path)
# and
@system_description.unmanaged_files.export_files_as_tarballs(destination)
```